frontend/src/components/profile/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import {Box, Container, Paper, Table, TableBody, Typography} from "@material-ui/core";
+import ProfileRow from "./profileRow";
+
+export default class Profile extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+
+        };
+    }
+
+    fetchProfile() {
+        // TODO: Get profile data from the backend here.
+    }
+
+    render() {
+        return (
+            <Box m={3}>
+                <Container maxWidth="md">
+                    <Typography variant="h4">
+                        <Box  my={2} ml={2} textAlign="left">Profile</Box>
+                    </Typography>
+
+                    <Table component={Paper} style={{tableLayout: "fixed"}}>
+                        <colgroup>
+                            <col style={{width: "22%"}}/>
+                            <col style={{width: "55%"}}/>
+                            <col style={{width: "25%"}}/>
```

Comment:
Prefer if this was in the CSS file for the application rather than as an inline style

Reply:
As said in #13, styling is another topic that we will want to tackle. I will want to improve this later on with a proper UI design.